service/src/main/java/org/sunbird/actor/otp/SendOTPActor.java

```diff
@@ -69,16 +68,16 @@ private void sendOTPViaEmail(
     emailTemplateMap.put(JsonKey.OTP, otp);
     emailTemplateMap.put(JsonKey.OTP_EXPIRATION_IN_MINUTES, OTPUtil.getOTPExpirationInMinutes());
     emailTemplateMap.put(JsonKey.TEMPLATE_ID, template);
-    Request emailRequest = null;
+    Request emailRequest;
     if (StringUtils.isBlank(otpType)) {
-      emailRequest = OTPUtil.sendOTPViaEmail(emailTemplateMap, context);
+      emailRequest = OTPUtil.getRequestToSendOTPViaEmail(emailTemplateMap, context);
     } else {
-      emailRequest = OTPUtil.sendOTPViaEmail(emailTemplateMap, RESET_PASSWORD, context);
+      emailRequest = OTPUtil.getRequestToSendOTPViaEmail(emailTemplateMap, JsonKey.RESET_PASSWORD, context);
     }
     emailRequest.setRequestContext(context);
     logger.info(
         context,
-        "SendOTPActor:sendOTP : Calling EmailServiceActor for Key = "
+        "SendOTPActor:sendOTPViaEmail : Calling EmailServiceActor for Key = "
             + logMaskService.maskEmail(key));
     tellToAnother(emailRequest);
```

Comment:
email is going through brackground actor, but not SMS. should we create a ticket to chnge this?